my-app/src/Logic.js

```diff
@@ -42,49 +43,84 @@ return checked;
 
 }
 
+function score(state, erase){
+  state.whiteIsNext
```

Comment:
I recommend using `if/else` blocks for anything more complex than assignment - this construct with the ternary operator introduces significant cognitive load to parse, as compared to the equivalent `if/else` block.